compliance/collection/auditlog/auditlog_impl.go

```diff
@@ -25,10 +25,24 @@ var (
 	stagesAllowList = set.NewFrozenStringSet("ResponseComplete", "Panic")
 
 	// resourceTypesAllowList is set of resources that will be sent.
-	resourceTypesAllowList = set.NewFrozenStringSet("secrets", "configmaps")
+	resourceTypesAllowList = set.NewFrozenStringSet("secrets", "configmaps", "clusterrolebindings", "clusterroles", "networkpolicies", "securitycontextconstraints", "egressfirewalls")
```

Comment:
add a test that verifies that for every entry in `resourceTypesAllowList` there is an entry in `auditResourceToKubeResource`